Index: src/main/java/org/example/Opgave4/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package org.example.Opgave4;\n\npublic class Main {\n\n    public static void main(String[] args) {\n        TextUI textUI = new TextUI();\n        textUI.play();\n    }\n}\n\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/example/Opgave4/Main.java b/src/main/java/org/example/Opgave4/Main.java
--- a/src/main/java/org/example/Opgave4/Main.java	(revision 0ceb2d6461bcbd7fe75a0b75c5028a6e9a964c8d)
+++ b/src/main/java/org/example/Opgave4/Main.java	(date 1703917859358)
@@ -3,8 +3,14 @@
 public class Main {
 
     public static void main(String[] args) {
+        /*
         TextUI textUI = new TextUI();
         textUI.play();
+
+         */
+        Loops loops = new Loops();
+        loops.drawCircle(10);
+
     }
 }
 
